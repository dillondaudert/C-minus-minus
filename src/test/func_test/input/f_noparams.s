	.section	.rodata
.int_wformat: .string "%d\n"
.str_wformat: .string "%s\n"
.int_rformat: .string "%d"
	.globl b1
	.type b1 @function
	.globl b2
	.type b2 @function
	.globl b3
	.type b3 @function
	.globl b4
	.type b4 @function
	.globl main
	.type main @function

b1:	nop
	pushq	%rbp
	movq	%rsp, %rbp
b2:	nop
	pushq	%rbp
	movq	%rsp, %rbp
b3:	nop
	pushq	%rbp
	movq	%rsp, %rbp
b4:	nop
	pushq	%rbp
	movq	%rsp, %rbp
main:	nop
	pushq	%rbp
	movq	%rsp, %rbp
	ret
	ret
	ret
	ret
	leave
	ret
	.size	b1, .-b1
	.size	b2, .-b2
	.size	b3, .-b3
	.size	b4, .-b4
	.size	main, .-main

