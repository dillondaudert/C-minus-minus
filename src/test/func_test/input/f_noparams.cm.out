st_init created the symbol table hash map
Adding new symbol: b1, b1, offset: 0, type: 2, size: 4, arr: 0
st_add added b1 at position 48
Added process b1 to stable





Adding new symbol: b2, b2, offset: 0, type: 2, size: 4, arr: 0
st_add added b2 at position 249
Added process b2 to stable





Adding new symbol: b3, b3, offset: 0, type: 2, size: 4, arr: 0
st_add added b3 at position 182
Added process b3 to stable





Adding new symbol: b4, b4, offset: 0, type: 2, size: 4, arr: 0
st_add added b4 at position 383
Added process b4 to stable





Adding new symbol: main, main, offset: 0, type: 2, size: 4, arr: 0
st_add added main at position 0
Added process main to stable




Freed symbol at position 0
Freed symbol at position 48
Freed symbol at position 182
Freed symbol at position 249
Freed symbol at position 383
Freed symbol table
